doc/Pull Request Acceptance Checklist.md

```diff
@@ -0,0 +1,15 @@
+# Pull Request Acceptance Checklist
+
+## Beagle Project
+ * detailed expanatory Javadoc
+ * No checkstyle violations (errors or warnings)
+ * 100% Code coverage by JUnit Tests
+ * JUnit test check were created by another person and are complete (check everything possible)
```

Comment:
Well, what do you mean by “check everything one can check”, if a method takes a `long`, do you want to check all 2^64 possible input values?